cfgov/legacy/housing_counselor/geocoder.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import, unicode_literals
+
+from six import print_
+
+
+def geocode_counselors(counselors, **kwargs):
+    """Fill in missing latitude/longitude data for housing counselors."""
+    return ZipCodeBasedGeocoder(**kwargs).geocode(counselors)
+
+
+class ZipCodeBasedGeocoder(object):
+    """Fill in missing latitude/longitude data using zipcode locations.
```

Comment:
That would be fine--  we should maybe open a platform issue that follows https://github.com/cfpb/django-hud/issues/20 . I think it *is* something we need to fix, but we should see what the product owners think.